Makefile

```diff
@@ -10,6 +11,7 @@ endef
 
 define make_docker
 	docker build --build-arg SVC_NAME=$(subst docker_,,$(1)) --tag=mainflux/$(subst docker_,,$(1)) -f docker/Dockerfile .
+	cd ./dashflux && docker build --build-arg SVC_NAME=$(DASHFLUX) --tag=mainflux/$(DASHFLUX) -f ./docker/Dockerfile .
```

Comment:
Dashflux `Dockerfile` does not use build arguments. Please remove it. 

Also, defining it inside `make_docker` will cause it to be built for every single Mainflux service, which is not good. In order to avoid it and achieve the desired behaviour, please move this line (without build args) to `dockers` task. Note that you don't need to change directory, just edit paths properly.